src/libraries/Microsoft.PowerFx.Interpreter/EvalVisitor.cs

```diff
@@ -257,6 +257,10 @@ public override async ValueTask<FormulaValue> Visit(CallNode node, EvalVisitorCo
             {
                 result = await asyncFunc.InvokeAsync(args, _cancellationToken).ConfigureAwait(false);
             }
+            else if (func is IAsyncTexlFunction2 asyncFunc2)
+            {
+                result = await asyncFunc2.InvokeAsync(this, args).ConfigureAwait(false);

```

Comment:
>result = await asyncFunc2.InvokeAsync(this, args).ConfigureAwait(false); [](http://example.com/codeflow?start=16&length=72)

New interface 
Notice the lack of CancellationToken here as we have it in the runtime context

Reply:
Removed CancellationToken from it